diff -ur og-0.21.0-o/lib/og/errors.rb og-0.21.0/lib/og/errors.rb
--- og-0.21.0-o/lib/og/errors.rb	2005-07-25 00:54:35.000000000 -0600
+++ og-0.21.0/lib/og/errors.rb	2005-07-25 08:33:38.000000000 -0600
@@ -11,6 +11,10 @@
   def initialize(original_exception, info = nil)
     @original_exception, @info = original_exception, info
   end
+
+	def message
+		super + '(' + info + ')'
+	end
 end
 
 end
Only in og-0.21.0/lib/og: errors.rb.orig
Only in og-0.21.0/lib/og: errors.rb.rej
